css/styles.css

```
@@ -259,13 +265,165 @@ body {
   text-decoration: none;
 }
 
+.social-media__icon {
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+}
+
+.social-media__local,
+.social-media__gitblog,
+.social-media__git,
+.social-media__item--disabled {

```

Acho que aqui tu poderia criar um CSS base para essas classes. `.social-nedia__box` talvez?



```suggestion
.social-nedia__box {
```